src/Documents/Queries/MoreLikeThis/MoreLikeThisStopWords.ts

```diff
@@ -1,7 +1,12 @@
 
 import {SetupDocumentBase} from "../../SetupDocumentBase";
+import {ObjectUtil} from "../../../Utility/ObjectUtil";
 
 export class MoreLikeThisStopWords extends SetupDocumentBase {
     public id: string;
     public stopWords: string[];
-}
\ No newline at end of file
+
+    public toRemoteFieldNames() {
+        return ObjectUtil.transformObjectKeys(this, { defaultTransform: "pascal" });
```

Comment:
Can't recall if `transformObjectKeys()` method is mutatses the object or returning a new one ?

Reply:
looks like it creates new object.